cfgov/jinja2/v1/index.html

```diff
@@ -13,64 +13,500 @@
-                            <li class="list_item">
-                                <a class="list_link jump-link jump-link__external-link"
-                                   href="https://github.com/cfpb/cfgov-refresh">
-                                    See the code for this redesign on GitHub
-                                </a>
-                            </li>
-                        </ul>
+                                <ul class="list list__links">
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Link Text
+                                        </a>
+                                    </li>
+                                    <li class="list_item">
+                                        <a class="list_link"
+                                           href="#">
+                                            Link Text
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
                     </section>
-                </div><!-- END .content-l -->
-            </div><!-- END .content_main -->
-        </div><!-- END .content_wrapper -->
+                    #}
+                </div>
+            </div>
+        </div>
+
+        <div class="o-well
+                    o-well__home" >
+            <h2>Standing up for you</h2>
+            <div class="content-l
+                        content-l__large-gutters">
+                <section class="content-l_col
+                                content-l_col-1-2">
+                    <h3>We hold companies accountable for illegal practices</h3>
+                    <div class="content-l">
+                        <div class="o-card">
+                            <div class="m-card">
+                                <h3 class="h1 m-card_heading">$11.2 billion in relief</h3>
+                                <p>Amount companies must provide harmed consumers</p>
+                            </div>
+                        </div>
+                        <div class="o-card">
+                            <div class="m-card">
+                                <h3 class="h1 m-card_heading">25 million consumers</h3>
+                                <p>Number of people receiving relief because of our actions</p>
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <section class="content-l_col
+                                content-l_col-1-2
+                                content-l_col__before-divider">
+                    <h3>We listen to consumers and make their voices heard</h3>
+                    <div class="content-l">
+                        <div class="o-card">
```

Comment:
Will `o-card` ever contain more than one `m-card`? Should this perhaps be `m-card` and the `m-card` below should be `m-card_content` or `m-card_wrapper`?